pkg/pillar/cmd/ledmanager/ledmanager.go

```diff
@@ -204,25 +209,36 @@ func Run(ps *pubsub.PubSub, loggerArg *logrus.Logger, logArg *base.LogObject) in
 	ps.StillRunning(agentName, warningTime, errorTime)
 
 	model := hardware.GetHardwareModel(log)
-	log.Functionf("Got HardwareModel %s", model)
+	log.Warningf("Got HardwareModel %s", model)
```

Comment:
Why a warning? Log.Noticef should suffice to have it be sent to the controller log API by default.

Reply:
Corrected, thanks